ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -87,7 +87,9 @@
 
     private static final By CONDITIONS_LINK = By
             .cssSelector(".conditions .info-header i.right");
-
+	
+	private static final By ATTACHMENTS_LINK = By.id("attachments.attachments.visitActions.default");
```

Comment:
@kdaud thanks for working on this. around here are you trying to use the link Id or the link Text?

Reply:
> @kdaud thanks for working on this. around here are you trying to use the link Id or the link Text?

Am retrieving the element by its  id